src/video/VDP.cc

```diff
@@ -1682,6 +1685,47 @@ void VDP::VRAMPointerDebug::write(unsigned address, byte value, EmuTime::param /
 	}
 }
 
+// class RegisterLatchStatusDebug
+
+VDP::RegisterLatchStatusDebug::RegisterLatchStatusDebug(VDP &vdp_)
+	: SimpleDebuggable(vdp_.getMotherBoard(),
+			vdp_.getName() + " register latch status", "V99x8 register latch status (0 = value, 1 = register)", 1)
```

Comment:
Detail: I understand what this means. But for occasional users it might be ambiguous whether value/register means the thing that was last written, or the thing that is expected to be written next.

Same for RB/G below.

Reply:
Should I describe it more or less in this line: `v99x8 register latch status (0 = empty, 1 = stored value)`?

Reply:
And `v99x8 palette latch status (0 = expected red & blue components, 1 = expected green component)`